### PR TITLE
EVA-3100 count with alleles_match=false

### PR DESCRIPTION
**Pull request by nitin-ebi**

### CODE REVIEW

File: tasks/eva_3100/alleles_match_count.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                ss_acc_list = [int(line.split(",")[1].strip()) for line in list(islice(sve_file, 100000))]
+                logger.info(f'Processing Split Batch Id {batch_id}')
+                batch_id = batch_id + 1
+                if not ss_acc_list:
+                    break
+                filter_criteria = {'eventType': 'SS_SPLIT', 'accession': {'$in': ss_acc_list}}
+                records = find_documents(mongo_source, DBSNP_SUBMITTED_VARIANT_OPERATION_ENTITY, filter_criteria)
+                for record in records:
+                    event_id = record['_id']
+                    split_into_ss_accession = record['accession']
```
Comment (sundarvenkata-EBI) [inline tasks/eva_3100/alleles_match_count.py:120]:
```suggestion
                    split_from_ss_accession = record['accession']
```

**Review by tcezard — APPROVED** [review]

**Comment by sundarvenkata-EBI** [comment]
@nitin-ebi you mentioned that you made some changes to the scripts. Not sure if they have been incorporated into the PR...

File: tasks/eva_3100/alleles_match_count.py (hunk 1 above)
Comment (nitin-ebi) [reply tasks/eva_3100/alleles_match_count.py]: done

**Comment by nitin-ebi** [comment]
> @nitin-ebi you mentioned that you made some changes to the scripts. Not sure if they have been incorporated into the PR...

updated the PR now

**Review by sundarvenkata-EBI — APPROVED** [review]

**Pull request merged**